src/com/twitter/intellij/pants/service/PantsCompileOptionsExecutor.java

```diff
@@ -196,8 +196,22 @@ private ProcessOutput getProcessOutput(
     return processOutput;
   }
 
+  private boolean isAtLeastPantsVersion(int major, int minor) throws ExecutionException {
+    String pantsVersion = getPantsVersion();
+    String[] splitVersion = pantsVersion.trim().split("\\.");
+    return splitVersion.length >= 3 && Integer.parseInt(splitVersion[0]) >= major && Integer.parseInt(splitVersion[1]) >= minor;
+  }
```

Comment:
Could you make this a wrapper function of https://github.com/pantsbuild/intellij-pants-plugin/blob/e74ce17b082979b9d0982956ed5ef58b9f5a4985/common/com/twitter/intellij/pants/util/PantsUtil.java#L916, or possibly directly use the other instead?

Reply:
Fixed! I added a new method `isCompatibleVersion` - it's a bit easier to reason about than when returning int I think :thinking: